strnstr.cpp

```diff
@@ -18,7 +18,7 @@ unsigned char *strnstr(unsigned char *str1, const char *str2, uint16_t size) {
     if(ch == str2[0] && a+len <= size) {
       c = a;
       a++;
-      for(b=1;b<=len;a++, b++) {
+      for(b=1;b<len;a++, b++) {
```

Comment:
As explained earlier this one is needed to be able to check a str2 which is found on the end of str1. Without the change the for loop will overflow beyond the boundary of str2 and str1.

Reply:
Sorry, i meant the changes inside webserver.cpp. I have confirmed this one myself.